deepmd/utils/argcheck_mt.py

```diff
@@ -0,0 +1,685 @@
+from dargs import dargs, Argument, Variant
+from deepmd.common import ACTIVATION_FN_DICT, PRECISION_DICT
+
+
+def list_to_doc(xx):
+    items = []
+    for ii in xx:
+        if len(items) == 0:
+            items.append(f'"{ii}"')
+        else:
+            items.append(f', "{ii}"')
+    items.append('.')
+    return ''.join(items)
+
+
+def make_link(content, ref_key):
+    return f'`{content} <{ref_key}_>`_' if not dargs.RAW_ANCHOR \
+        else f'`{content} <#{ref_key}>`_'
+
+
+def type_embedding_args():
+    doc_neuron = 'Number of neurons in each hidden layers of the embedding net. When two layers are of the same size or one layer is twice as large as the previous layer, a skip connection is built.'
+    doc_resnet_dt = 'Whether to use a "Timestep" in the skip connection'
+    doc_seed = 'Random seed for parameter initialization'
+    doc_activation_function = f'The activation function in the embedding net. Supported activation functions are {list_to_doc(ACTIVATION_FN_DICT.keys())}'
+    doc_precision = f'The precision of the embedding net parameters, supported options are {list_to_doc(PRECISION_DICT.keys())}'
+    doc_trainable = 'If the parameters in the embedding net are trainable'
+    
+    return [
+        Argument("neuron", list, optional = True, default = [2, 4, 8], doc = doc_neuron),
+        Argument("activation_function", str, optional = True, default = 'tanh', doc = doc_activation_function),
+        Argument("resnet_dt", bool, optional = True, default = False, doc = doc_resnet_dt),
+        Argument("precision", str, optional = True, default = "float64", doc = doc_precision),
+        Argument("trainable", bool, optional = True, default = True, doc = doc_trainable),
+        Argument("seed", [int,None], optional = True, doc = doc_seed),
+    ]        
+
+
+#  --- Descriptor configurations: --- #
+def descrpt_local_frame_args ():
```

Comment:
I don't think most of these methods are different from those in `argcheck.py`. It's silly to copy them again.